pointcloud/pointcloud_utils_test.go

```diff
@@ -49,12 +49,13 @@ func TestBoundingBoxFromPointCloud(t *testing.T) {
 	}
 
 	for _, c := range cases {
-		expectedBox, err := spatialmath.NewBox(spatialmath.NewPoseFromPoint(c.expectedCenter), c.expectedDims)
+		expectedBox, err := spatialmath.NewBox(spatialmath.NewPoseFromPoint(c.expectedCenter), c.expectedDims, "")
 		test.That(t, err, test.ShouldBeNil)
-		box, err := BoundingBoxFromPointCloud(c.pc)
+		box, err := BoundingBoxFromPointCloudWithLabel(c.pc, "box")
```

Comment:
Updated this test to test the new `BoundingBoxFromPointCloudWithLabel` function, which is actually the old function's logic with a new function signature (it takes in a label now). It seemed liked the simplest way to cover the new function, `BoundingBoxFromPointCloudWithLabel`. See GitHub comment above this function for more info.